Source/ASCollectionView.mm

```diff
@@ -317,12 +317,7 @@ - (instancetype)_initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionV
   
   [self _configureCollectionViewLayout:layout];
   
-  if (ASActivateExperimentalFeature(ASExperimentalNewDefaultCellLayoutMode)) {
-    _cellLayoutMode = ASCellLayoutModeSyncForSmallContent;
-  } else {
-    _cellLayoutMode = ASCellLayoutModeNone;
-  }
-  
+  _cellLayoutMode = ASCellLayoutModeSyncForSmallContent;
```

Comment:
This one can go away as the default one will be `ASCellLayoutModeNone`. In fact the `ASCellLayoutModeSyncForSmallContent` is not needed at all anymore